welcome/about-kueue.adoc

```diff
@@ -0,0 +1,53 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="about-kueue"]
+= About {product-title}
+:context: about-kueue
+
+toc::[]
+
+{product-title} is a Kubernetes-native system that manages access to resources for jobs.
```

Comment:
```suggestion
{product-title} is a Kubernetes-native system that manages access to resources for workloads.
```

Reply:
Discussed offline and decided to agree on "jobs" since the note explaining this is included. We will include more details about different job types as docs are added for other features.